turbinia/evidence.py

```diff
@@ -978,18 +1061,22 @@ class EwfDisk(Evidence):
   """Evidence object for a EWF based evidence.
 
   Attributes:
-    device_path (str): Path to the mounted loop device.
+    source_path (str): Path to the mounted loop device.
```

Comment:
Minor nit, but I think we want to keep `device_path` in the attribute list too since it's used by this object, right?

Reply:
Looked at the two methods in ```mount_local``` for this evidence type and they actually don't use ```device_path``` anywhere, so I figured we can just use source_path for consistency since that's what the base class provides and that's also used by ```RawDisk``` to identify the path to the evidence file(s).

In fact, we can probably just remove ```device_path``` references in this object, unless I missed something?

Reply:
Fixed docstring. Thanks for the additional context on device_path usage.